types/src/front/user.ts

```diff
@@ -31,6 +31,8 @@ export type LightWorkspaceType = {
   name: string;
   role: RoleType;
   segmentation: WorkspaceSegmentationType;
+  whiteListedProviders: string[] | null;
+  defaultEmbeddingProvider: string | null;
```

Comment:
Seems like we can have `string` as a type here no?

Reply:
We can likely type it more as well. Given that you typed the model you can likely type this as well right?

Reply:
You're right, will do 👍🏼